MANIFEST.in

```diff
@@ -3,10 +3,18 @@ include reqs/*.txt
 include LICENSE
 include great_expectations/data_context/checkpoint_template.yml
 include great_expectations/init_notebooks/*/*.ipynb
+include versioneer.py
+include great_expectations/_version.py
+include versioneer.cfg
+include py.typed
+
 recursive-include great_expectations/render *.j2 *.md *.py
+recursive-include great_expectations *.py
 recursive-include great_expectations *.pyi
+recursive-include great_expectations *.typed
+recursive-include great_expectations py.typed
```

Comment:
We were missing including `py.typed` in this file, but this would only affect the `.tar.gz` distribution, not the `.whl`.